docs/schedule/validate.ko.md

```diff
@@ -0,0 +1,27 @@
+# GTFS Schedule 유효성 검사기
+
+이 무료 오픈 소스 정식GTFS Schedule 유효성 검사기는 [MobilityData](https://mobilitydata.org) 의 도움을 받아 유지 관리됩니다.GTFS 지역 사회. 이를 사용하여 공식 데이터 세트와 비교하여 데이터 세트를 평가합니다.GTFS [참조](reference.md) 및 [모범 사례](best-practices.md) . 브라우저에서 열고 다른 당사자와 공유할 수 있는 HTML 형식의 사용하기 쉬운 검증 보고서를 제공합니다.
+
+<!-- <img class="center" src="../../assets/validator_animation.gif" width="150"> -->
+
+<br/>
+
+<div class="usage-buttons"> <a class="button" href="https://share.mobilitydata.org/validator-installer-mac">Mac용 다운로드<img class="icon" src="../../assets/apple.svg" width="20"/></a> <a class="button" href="https://share.mobilitydata.org/validator-installer-windows">Windows용 다운로드<img class="icon" src="../../assets/windows.svg" width="20"/></a> <a class="button" href="https://share.mobilitydata.org/validator-installer-ubuntu">우분투용 다운로드<img class="icon" src="../../assets/ubuntu.svg" width="20"/></a></div>
+
+**버전 v4.0.0** / [릴리스 노트 읽기](https://github.com/MobilityData/gtfs-validator/releases/latest)
+
+<hr/>
+
+## 용법
+
+<div class="usage"><div class="usage-list"><ol><li> 운영 체제에 따라 최신 버전의 프로젝트를 다운로드하십시오.</li><li> 설치가 완료되면 응용 프로그램을 실행하면 창이 나타납니다.</li><li> 로드GTFS 데이터 세트: 다음을 지정할 수 있습니다.url , ZIP 파일 또는 피드의 개별 .txt 파일이 포함된 디렉토리입니다.</li><li> 출력 디렉터리 선택: 유효성 검사 보고서가 작성될 디렉터리입니다.</li><li> 유효성 검사가 완료되면 결과가 브라우저에서 자동으로 열립니다. 표시되지 않으면 출력 경로에 생성된 report.html 파일을 열면 됩니다.</li><li> 유효성 검사기가 데이터에서 문제를 발견했는지 확인하고 문제를 해결하는 방법에 대한 설명서 링크를 확인할 수 있습니다. 유효성 검사 보고서는 HTML 파일로 저장되므로 쉽게 저장하고 다른 사람과 공유할 수 있습니다. </li></ol></div><div class="usage-video"><video class="center" width="560" height="315" controls=""><source src="../../assets/validator_demo_large.mp4" type="video/mp4"/></video></div></div>
+
+## 이 유효성 검사기를 데이터 파이프라인의 일부로 사용
+
+데이터 파이프라인에서 이 도구를 사용하는 방법에 대한 자세한 지침을 보려면 [를 방문하세요 저장소](https://github.com/MobilityData/gtfs-validator) .
+
+## 기여
+
+이 프로젝트는 커뮤니티의 노력이며 이 프로젝트에 관심이 있는 사람은 누구나 커뮤니티에 가입하고 의사 결정에 참여하며 다양한 방식으로 프로젝트를 진행하는 데 도움을 줄 수 있습니다. 기여할 수 있는 다양한 방법이 있습니다: 새로운 기능에 대한 아이디어 공유, 문제 보고, 문서 개선, 다른 사람에게 사용 방법 교육 등.
+
+이 프로젝트에 기여하는 데 관심이 있으십니까? [를 방문하십시오 GitHub 저장소](https://github.com/MobilityData/gtfs-validator) 및 [읽기 기여 가이드라인](https://github.com/MobilityData/gtfs-validator/blob/master/docs/CONTRIBUTING.md) .
```

Comment:
this one is wrong, assuming the former since there is not mention of Github in the hyperlink